guides/common/modules/proc_using-project-as-a-source-of-content-for-imagebuilder.adoc

```diff
@@ -0,0 +1,21 @@
+[id="using-project-as-a-source-of-content-for-imagebuilder"]
+= Using {Project} as a source of content for {LoraxCompose}
+
+In {Project}, you can integrate with {Cockpit}.
+By using {Cockpit}, you can access {LoraxCompose} and build images.
+When you configure {LoraxCompose} for image building, you can use {Project} content to build the images.
+
+.Prerequisites
+* Ensure you have imported content into {ProjectServer}.
+For more information, see xref:Importing_Content_{context}[].
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Products*, select your Product and click the repository you require.
+Locate the *Published at* field and note the URL.
+. If your repository is published at a secured URL (HTTPS) only, ensure you have configured the Red{nbsp}Hat Subscription Manager on your {LoraxCompose} host machine to use the Katello server CA certificate.
+For more information, see link:https://access.redhat.com/solutions/7004689[Composer Image Builder fails when multiple custom repos are defined on the Satellite] in the _Red Hat Knowledgebase_.
+. link:{RHELDocsBaseURL}8/html/composing_a_customized_rhel_system_image/managing-repositories_composing-a-customized-rhel-system-image#overriding-a-system-repository_managing-repositories[Override the required system repository in {LoraxCompose} configuration] and use the URL of your {Project} repository as a `baseurl`.
```

Comment:
I'm not sure if it requires subscriptions. Probably depends on the repository...